source/MaterialXGenShader/HwShaderGenerator.cpp

```diff
@@ -19,11 +19,6 @@ void HwShaderGenerator::bindLightShader(const NodeDef& nodeDef, size_t lightType
         throw ExceptionShaderGenError("Error binding light shader. Given nodedef '" + nodeDef.getName() + "' is not of lightshader type");
     }
 
-    if (getBoundLightShader(lightTypeId))
```

Comment:
Previous logic was to add a "light type". Logic now is to add a light instance. We should not error when you try to add > 1 instance of the same light type.